testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -667,3 +667,424 @@ impl IdentityChangeDataSet {
             .expect("Can't parse the `/keys/upload` response")
     }
 }
+
+pub struct VerificationLatchTestData {}
+
+#[allow(dead_code)]
```

Comment:
Would `#[cfg(test)]` work? If so, it's better than allowing dead code.

Reply:
No it's not, if I add it then the crypto crate won't compile. If I understand well the `test` config doesn't propagate to other crates, so the annotated code here won't be accessible. I thinking that's why there is the `testing` cfg? Not sure how to make it work

Reply:
Yes, that is why the `testing` feature exists. It's painful, but better than `#[allow(dead_code)]`. There is an example in `crates/matrix-sdk-crypto/src/lib.rs` in the module called `testing`.

Reply:
There are other #[allow(dead_code)] in this file unrelated to the current changes. Will do a follow-up PR to remove them